probe/endpoint/reporter.go

```diff
@@ -89,6 +89,10 @@ type fourTuple struct {
 	fromPort, toPort uint16
 }
 
+func (t fourTuple) String() string {
```

Comment:
I was following the Golang "Stringer" interface:
https://golang.org/pkg/fmt/#Stringer